osu.Game/Screens/Select/DifficultyRecommender.cs

```diff
@@ -75,12 +89,35 @@ private void calculateRecommendedDifficulties()
             });
         }
 
+        private IEnumerable<RulesetInfo> getBestRulesetOrder()
```

Comment:
what does "best" mean here?

Reply:
it means "best guess at what user wants to play". I'll add xmldoc to the function, but I can't come up with a better name.